lib/src/multires_stochastic_texture_synthesis.rs

```diff
@@ -981,8 +986,13 @@ impl Generator {
             })
             .unwrap();
 
-            // Some items might still be pending a resolve flush. Do it now before we start the next stage.
-            self.force_flush_resolved(is_tiling_mode);
+            {
+                // append all resolved threads to resolved
```

Comment:
Sounds like we're appending threads. Would rephrase to "append all per-thread resolved lists to the global list"

Reply:
haha true I'll rewrite that